step-definitions/Hotels/Sprint1SD.js

```diff
@@ -0,0 +1,305 @@
+const { Given, Then, When } = require("@wdio/cucumber-framework");
+const { expect } = require("chai");
+const HotelsFeedback = require("../../POM/Hotels/HotelsFeedback");
+const HotelsHome = require("../../POM/Hotels/HotelsHome");
+const HotelsSignIn = require("../../POM/Hotels/HotelsSignIn");
+const HotelsSignUp = require("../../POM/Hotels/HotelsSignUp");
+
+const hotelsHome = new HotelsHome
+const hotelsSignin = new HotelsSignIn
+const hotelsSignUp = new HotelsSignUp
+const hotelsFeedback = new HotelsFeedback
+
+
+
+// TC18
+
+Given(/^I am on Hotels landing page$/, async function() {
+    await browser.url('https://www.hotels.com/');
+    await browser.pause(3000);   
```

Comment:
browser.pause() not efficient at all. You can use waitFor() for waitUntil() instead.